OpenRA.Mods.D2k/Traits/MakesNoise.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	[Desc("This actor makes noise.")]
```

Comment:
The best way to describe it is probably to say "Does this actor attract worms?"

The idea of noise attracting worms is lore specific, and probably should not be used for any technical writing.

On 2015-01-08, at 1:53 PM, Taryn Hill <notifications@github.com> wrote:

> In OpenRA.Mods.D2k/Traits/MakesNoise.cs:
> 
> > @@ -0,0 +1,36 @@
> > +#region Copyright & License Information
> > +/*
> > +* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
> > +* This file is part of OpenRA, which is free software. It is made
> > +* available to you under the terms of the GNU General Public License
> > +* as published by the Free Software Foundation. For more information,
> > +* see COPYING.
> > +*/
> > +#endregion
> > +
> > +using OpenRA.Traits;
> > +
> > +namespace OpenRA.Mods.D2k.Traits
> > +{
> > +	[Desc("This actor makes noise.")]
> Noise is an in-game logic system (currently used by the D2k worms), not a way to play audio files.
> 
> I know it is rudimentary and not so nice, but it does explain it.
> Any better suggestions are welcomed.
> 
> \
> Reply to this email directly or view it on GitHub.
>